src/Core/WinSWCore/ServiceDescriptor.cs

```diff
@@ -355,6 +362,49 @@ public string LogMode
             }
         }
 
+        public string LogName

```

Comment:
Imho all the options should be defined in the `<logging>` tag

Reply:
I've followed the same logic as logmode, logpath, let me know if its wrong.

Reply:
ack, will need to think about it twice.